pages/c/_cluster/apps/install.vue

```diff
@@ -921,354 +762,300 @@ export default {
+            :name-required="false"
+            :name-ns-hidden="!showNameEditor"
+            :force-namespace="forceNamespace"
+            :namespace-new-allowed="!existing && !forceNamespace"
+            :extra-columns="showProject ? ['project'] : []"
           >
-            <template v-slot:option="opt">
-              <template v-if="opt.kind === 'divider'">
-                <hr />
-              </template>
-              <template v-else-if="opt.kind === 'label'">
-                <b style="position: relative; left: -2.5px;">{{ opt.label }}</b>
-              </template>
+            <template v-if="showProject" #project>
+              <LabeledSelect
+                v-model="project"
+                :disabled="!namespaceIsNew"
+                :label="t('catalog.install.project')"
+                option-key="id"
+                :options="projectOpts"
+                :tooltip="!namespaceIsNew ? t('catalog.install.namespaceIsInProject', {namespace: value.metadata.namespace}, true) : ''"
+                :hover-tooltip="!namespaceIsNew"
+                :status="'info'"
+              />
             </template>
-          </LabeledSelect>
+          </NameNsDescription>
         </div>
-        <div v-if="chart" class="col span-6">
-          <LabeledSelect
-            :label="t('catalog.install.version')"
-            :value="$route.query.version"
-            :options="filteredVersions"
-            :selectable="version => !version.disabled"
-            @input="selectVersion"
-          />
+      </template>
+      <template #helmValues>
+        <p v-if="step2Description" class="row mb-10">
+          {{ step2Description }}
+        </p>
+        <div class="step__values__controls">
+          <!-- Edit as YAML / Back to Form -->
+          <ButtonGroup
+            v-if="(valuesComponent || hasQuestions)"
+            v-model="preShowPreview"
+            :options="formYamlOptions"
+            inactive-class="bg-disabled btn-sm"
+            active-class="bg-primary btn-sm"
+            :disabled="preShowPreview != showPreview"
+          ></ButtonGroup>
+          <!-- Continue Editing / Show Diff -->
+          <ButtonGroup
+            v-if="showingYaml"
+            v-model="showDiff"
+            :options="editYamlOptions"
+            inactive-class="bg-disabled btn-sm"
+            active-class="bg-primary btn-sm"
+          ></ButtonGroup>
+          <div class="step__values__controls--spacer">
+&nbsp;
+          </div>
+          <ButtonGroup
+            v-if="showingYaml && showDiff"
+            v-model="diffMode"
+            :options="yamlDiffModeOptions"
+            inactive-class="bg-disabled btn-sm"
+            active-class="bg-primary btn-sm"
+          ></ButtonGroup>
+          <div v-if="hasReadme" class="btn-group">
+            <button type="button" class="btn bg-primary btn-sm" @click="showSlideIn = !showSlideIn">
+              {{ t('catalog.install.steps.helmValues.chartInfo.button') }}
```

Comment:
This is cool but quite narrow and the READMEs are usually full of quite wide tables, so it's hard to actually use.  The horizontal scrollbar also doesn't show up so it's not obvious there is even more to see.  And it obscures the next/done button.

I think this should add a "window manager" tab (like container shell/logs) instead.  Those are the established floating window component, are full-width instead of narrow, and you're going end up seeing it anyway when you launch the app because the output shows up there.

Reply:
There's now an option to 'dock' the sidebar into the windows manager in the slide out
![image](https://user-images.githubusercontent.com/18697775/118029602-3a375500-b35c-11eb-944f-b1712aca2bf1.png)

![image](https://user-images.githubusercontent.com/18697775/118029779-65ba3f80-b35c-11eb-8907-e386bdc2531c.png)

This should hopefully cover the best of all worlds.

Text and icon still need updating.

Reply:
The icon on this is the only thing left, I've reached out to @lvuch